x-pack/plugins/security_solution/common/endpoint/index_data.ts

```diff
@@ -91,7 +93,7 @@ export async function indexHostsAndAlerts(
   const realPolicies: Record<string, CreatePackagePolicyResponse['item']> = {};
 
   for (let i = 0; i < numHosts; i++) {
-    const generator = new EndpointDocGenerator(random);
+    const generator = new (DocGenerator || EndpointDocGenerator)(random);
```

Comment:
since `DocGenerator` already defaults to `EndpointDocGenerator`

```suggestion
    const generator = new (DocGenerator)(random);
```

Reply:
Good point. Applied 🙂